CMakeLists.txt

```diff
@@ -36,7 +36,7 @@ set(CMAKE_VERBOSE_MAKEFILE ON)
 if (UNIX)
     # Linux and OSX
     set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -g -fPIC -Werror")
-    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++0x -std=gnu++0x")
+    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++0x -std=gnu++0x -fPIC")
```

Comment:
@howryu @notkriswagner do you have any specific reason why you didn't add -fPIC? Without this, the PHP PDO build will fail.

Reply:
Go ahead and add it. No reasons for me.